hlib/python/hlib/frontend/relay_parser.py

```diff
@@ -0,0 +1,696 @@
+import tvm
+import keras
+import tvm.relay.frontend as relay_front
+import numpy as np
+import heterocl as hcl
+import hlib
+import re
+from ..utils import *
+from .relay_attr import _attrib, _convert_map
+from tvm.relay.expr import Function, Var, Call, Let, If, Constant
+from tvm.relay.expr import TupleGetItem, Tuple
+from tvm.relay.ty import TensorType, TupleType
+
+hcl.init(hcl.Float())
```

Comment:
Is this needed?

Reply:
hcl.init(hcl.Float()) isn't. I'll remove that